Structure_Engine/Query/Extrude.cs

```diff
@@ -38,6 +42,10 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Generates a extrusion of the section along the bar centreline.")]
+        [Input("bar", "The bar to get an extruded shape from. This required the proeprty of the bar do be of a type with a section profile.")]
```

Comment:
The Bar to get an extruded shape from. This requires the property of the Bar to be of a type with a section profile.